src/App.tsx

```diff
@@ -1,29 +1,56 @@
-import { Component } from 'react';
+/* eslint-disable no-console */
+/* eslint-disable consistent-return */
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
+import { getMovie } from './api/getMovie';
 
-interface State {
-  movies: Movie[];
-}
+export const App: React.FC = () => {
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [isMovieFound, setIsMovieFound] = useState(true);
+  const [foundMovie, setFoundMovie] = useState<Movie | null>(null);
```

Comment:
so you don't need boolean value here also
```suggestion
  const [foundMovie, setFoundMovie] = useState<Movie | null>(null);
```